src/updater.h

```diff
@@ -38,8 +38,11 @@ class MetadataUpdater {
     ResourceMetadata(const std::vector<std::string>& ids,
                      const MonitoredResource& resource,
                      MetadataStore::Metadata&& metadata)
-        : ids_(ids), resource_(resource),
-          metadata_(std::move(metadata)) {}
+        : ids_(ids), resource_(resource), metadata_(std::move(metadata)) {}
+    ResourceMetadata(const std::vector<std::string>& ids,
```

Comment:
This is the same as `ResourceMetadata(ids, resource, metadata.Clone())`. Why do you need this?

Reply:
fixed the kubernetes code to use metadata.Clone()